### PR TITLE
Feature/azure-k8n-storage

### PR DESCRIPTION
**Pull request by seriva**
- Added shared storage for k8n

### CODE REVIEW

File: core/src/epicli/data/azure/defaults/infrastructure/storage-share.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+kind: infrastructure/storage-share
+version: 0.3.0
```
Comment (lukurde) [inline core/src/epicli/data/azure/defaults/infrastructure/storage-share.yml]: Is that correct? Other versions in this PR were bumped to 0.4.0

File: core/src/epicli/data/common/ansible/playbooks/roles/kubernetes_master/templates/azure/k8s-persistent-volume.yml.j2 [hunk 2]
```diff
@@ -1,10 +1,14 @@
...
-  name: "pv0001" 
+metadata:
+  name: {{ specification.storage.name }}
+  labels:
+    name: {{ specification.storage.name }}
 spec:
+  storageClassName: azurefile
   capacity:
-    storage: "15Gi" 
+    storage: {{ specification.storage.capacity }}Gi
```
Comment (lukurde) [inline core/src/epicli/data/common/ansible/playbooks/roles/kubernetes_master/templates/azure/k8s-persistent-volume.yml.j2]: Are we sure we want the suffix hard-coded to Gibibytes?

File: core/src/epicli/data/azure/defaults/infrastructure/storage-share.yml (hunk 1 above)
Comment (seriva) [reply core/src/epicli/data/azure/defaults/infrastructure/storage-share.yml]: Fixed

File: core/src/epicli/data/common/ansible/playbooks/roles/kubernetes_master/templates/azure/k8s-persistent-volume.yml.j2 (hunk 2 above)
Comment (seriva) [reply core/src/epicli/data/common/ansible/playbooks/roles/kubernetes_master/templates/azure/k8s-persistent-volume.yml.j2]: As per Tomasz request yes.

Comment (lukurde) [reply core/src/epicli/data/common/ansible/playbooks/roles/kubernetes_master/templates/azure/k8s-persistent-volume.yml.j2]: Ok, then

**Pull request merged**